src/clr-core/alert/_properties.scss

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@include exports('alert.properties') {
+  @if $clr-use-custom-properties == true {
+    :root,
+    :host {
+      // Usage: ./_alert.clarity.scss
+      // Used for 'toggle' button for dropdowns on the right side of an alert
+      --clr-alert-action-color: var(--clr-color-neutral-700);
+      --clr-alert-action-active-color: var(--clr-color-secondary-action-900);
```

Comment:
The refactor that is on master was three parts, I am happy to pair up on this if you would like. The other components are 95% updated I am working on the last little bit.

1. Keep existing css props in clr-ui/clr-angular and no longer use them directly in core
2. Update to naming pattern removing the clr prefix and preventing global collisions
    `--[optional template element]-[css property]-[optional element state]: value;`
3. Anything using a global `--clr` var has a sass var fallback to keep component working when no global styles are present. (Storybook now will catch this for us since it loads components in isolation)

So for alerts I think it would look something like this

```scss
// alert.element.scss
:host {
  --color: var(--clr-color-neutral-700, #{$clr-color-neutral-700});
  --background: var(--clr-color-action-50, #{$clr-color-action-50});
  --border-radius: var(--clr-global-borderradius, #{$clr-global-borderradius});
  --border-color: var(--clr-color-action-400, #{$clr-color-action-400});
  --icon-color: var(--clr-color-action-600, #{$clr-color-action-600});
  --close-icon-color: var(--clr-color-neutral-700, #{$clr-color-neutral-700});
  --close-icon-color-hover: var(--clr-color-neutral-700, #{$clr-color-neutral-700});
}
```

Then altered states can follow the same pattern

```scss
:host([status='success']) {
  --background: var(--clr-color-success-50, #{$clr-color-success-50});
  --border-color: var(--clr-color-success-400, #{$clr-color-success-400});
  --icon-color: var(--clr-color-success-700, #{$clr-color-success-700});
}
```